requirements/base.txt

```diff
@@ -0,0 +1,8 @@
+boto3==1.3.1
+botocore==1.4.13
+docutils==0.12
```

Comment:
Which of these is actually used? I only see `boto3` and `botocore` directly addressed?